packages/atomic/src/components.d.ts

```diff
@@ -9853,19 +9830,6 @@ declare namespace LocalJSX {
          */
         "label": string;
     }
-    /**
-     * The `atomic-text` component leverages the I18n translation module through the atomic-search-interface.
-     */
-    interface AtomicText {
-        /**
-          * The count value used for plurals.
-         */
-        "count"?: number;
-        /**
-          * The string key value.
-         */
-        "value": string;
-    }
```

Comment:
What is this file event used for ? I can't remember.

Reply:
For syntaxic sugar iirc.
But this should be handled with https://github.com/coveo/ui-kit/pull/4787#discussion_r1883691965
i.e. the type would now be per component (then remashed together in the barrel/index files)